invenio_mail/__init__.py

```diff
@@ -52,6 +52,51 @@
 Content-Type: text/plain; charset="utf-8"...
 
 
+Using the API
+-------------
+
+A nice API let you create a message from a template, so you just have to
+give the rights arguments to get the full message. Moreover, it creates
+a complete e-mail with HTML message and text only message.
+
+To do so, you need to instantiate
+a ``invenio_mail.api.TemplatedMessage`` object, just like you
+would with a standard ``flask_mail.Message``:
```

Comment:
You need to configure intersphinx. See e.g. https://github.com/inveniosoftware/invenio-logging/blob/master/docs/conf.py#L331-L334